google/cloud/firestore_v1/order.py

```diff
@@ -50,7 +50,7 @@ def from_value(value):
         }
 
         if v not in lut:
-            raise ValueError("Could not detect value type for " + str(v))
+            raise ValueError(f"Could not detect value type for {v}")
```

Comment:
Believe it or not, it looks like this change broke [this test case](https://github.com/googleapis/python-firestore/pull/91/files#diff-a300827e49b735ab2c46f3124ea18bc2L192).

Reply:
FWIW, tests really shouldn't care about the text of an error message.